Application/Program.cs

```diff
@@ -0,0 +1,175 @@
+﻿using Simple_Inventory_Management_System.Domain;
+using Simple_Inventory_Management_System.Services;
+
+class Program
+{
+    static void AddNewProduct(InventoryService inventory)
+    {
+        string name ="";
```

Comment:
you can consider using var instead of using implicit references which might be better in some situations and will reduce redundant code sometimes. You can read more about the two ways. 

Apply this to all other applicable places.
```suggestion
        var name = string.Empty;
```